### DISCUSSION TITLE
PIA and Wireguard with port forwarding?

### DISCUSSION THREAD

**Discussion by Staticvi** [qdm12/gluetun#2077] [Q&A]
I am trying to set up Port Forwarding with Wireguard using PIA but all that is listed is OVPN settings. Does anyone know what I would put if I wanted to use Wireguard? PIA supports port forwarding with Wireguard on the desktop client.

**Comment by danutzzzzz** [comment]
Hi, ive just spent the last few hours trying to get this working with no joy, so i would also like to know who to get this configured.

**Comment by danutzzzzz** [comment]
@Staticvi i gave up waiting moved over to binhex
